conda/environments/streamz_dev.yml

```diff
@@ -6,6 +6,7 @@ channels:
 dependencies:
   - python>=3.7
   - pytest
+  - pytest-tornasync
```

Comment:
Do we need to add this dependency if because of the introduction of the `async` keyword?